src/node.cc

```diff
@@ -580,7 +580,7 @@ static struct {
 #endif  // __POSIX__
 
 
-inline void PlatformInit() {
+void PlatformInit() {
```

Comment:
`PlatformInit()` is kinda like an umbrella name that we didn't bother to give it some proper thought. Maybe rename it to something specific like `InitializeSignalHandlersAndStdio()`?